src/test/java/picard/sam/RevertSamTest.java

```diff
@@ -592,4 +604,135 @@ public void testHardClipRoundTrip() throws Exception {
             }
         }
     }
+
+
+    private final static boolean OUTPUT_BY_READ_GROUP = true;
+    // LOCAL_OUTPUT_MAP means the output map file itself is stored locally. The output SAM files will always be in the cloud.
+    private enum OutputMapOptions { LOCAL_OUTPUT_MAP, CLOUD_OUTPUT_MAP, NO_OUTPUT_MAP }
+
+    @DataProvider(name = "cloudTestData")
+    public Object[][] getCloudTestData() {
+        final PicardHtsPath tempCramOutputPath = PicardBucketUtils.getTempFilePath(GCloudTestUtils.TEST_OUTPUT_DEFAULT_STR, "RevertSam", ".cram");
```

Comment:
Since this is a cloud path, I would include that in the name to make it clear, i.e. tempCloudCram or something.

Reply:
done